cola/git.py

```diff
@@ -330,9 +332,14 @@ def git(self, cmd, *args, **kwargs):
                 # case of argv overflow. we should be safe from that but use
                 # defensive coding for the worst-case scenario. on other OS-en
                 # we have ENAMETOOLONG which doesn't exist in with32 API.
-                status, out, err = self.execute(['git', '--version'])
-                if status == 0:
-                    raise e
+                try:
+                    status, out, err = self.execute(['git', '--version'])
+                except FileNotFoundError:
+                    pass # git command is not available
```

Comment:
`FileNotFound` is a python3-ism; it's preferable to catch `OSError` to not break python2 users.  I'll merge this shortly with a fixup on top.  Thanks for the fix, indeed, this is a problem on all OSes.